packages/beacon-state-transition/src/allForks/util/epochContext.ts

```diff
@@ -325,6 +327,36 @@ export class EpochContext {
     return validatorIndices;
   }
 
+  getCommitteeAssignments(
+    epoch: Epoch,
+    validatorData: {pubkey: BLSPubkey; index: ValidatorIndex}[]
+  ): routes.validator.AttesterDuty[] {
```

Comment:
I don't think this is the return type we want here?

Or maybe we do. There's also the `CommitteeAssignment` type. Maybe a chance to just stop using this `CommitteeAssignment` type which is only used during intermediate steps.

Thoughts @dapplion ?

Reply:
yeah we can change the name of this function to `getAttesterDuties` or something like that, if that's better.